includes/Modules/Analytics.php

```diff
@@ -1272,7 +1281,12 @@ protected function has_access_to_property( $property_id, $accounts ) {
 		$response = false;
 
 		foreach ( $accounts as $account ) {
-			$account_id = $account->getId();
+			if ( is_array( $account ) ) {
+				$account_id = $account['id'];
+			} else {
+				$account_id = $account->getId();
+			}
```

Comment:
What's the reason this can be an array? Probably we should fix that at the root to make sure it's an object, unless that is out of our control.

Reply:
@felixarntz this is related to 'tag-permission' datapoint, as this is going to be moved to other issue I'll revert and add some context in following PR